src/main/java/org/craftercms/studio/impl/v1/service/site/SiteServiceImpl.java

```diff
@@ -744,19 +782,40 @@ private void createSiteCloneRemote(String siteId, String sandboxBranch, String d
 
                 upgradeManager.upgradeSite(siteId);
 
-                insertCreateSiteAuditLog(siteId, siteId);
-
                 // Add default groups
                 logger.info("Adding default groups for site " + siteId);
                 addDefaultGroupsForNewSite(siteId);
 
+                String lastCommitId = contentRepositoryV2.getRepoLastCommitId(siteId);
```

Comment:
Change to not create an audit log. Instead, set the markers such that it will calculate from the beginning.